README.md

```diff
@@ -270,6 +270,28 @@ Configure ccache (installed by kpatch package):
 
     ccache --max-size=5G
 
+#### Gentoo
+
+*NOTE: You'll need about 15GB of free disk space for the kpatch-build cache in
+`~/.kpatch` and for ccache.*
+
+Install Kpatch and Kpatch dependencies:
+
+```bash
+emerge --ask sys-kernel/kpatch
+```
+
+Install ccache (optional):
+
+```bash
+emerge --ask dev-util/ccache
+```
+
+Configure ccache:
+
+```bash
+ccache --max-size=5G
+```
 
```

Comment:
Does `kpatch-build` work out of the box for Gentoo?  There's a bunch of distro-specific code there to fetch proper kernel sources and whatnot.

Reply:
yes, it work out of the box but only by using -s for specifying the source directory.
Of course -r dosen't work.

Reply:
Okay, good.  How about the kernel sources on the user's machine?  Are kernel sources always available on a Gentoo install somewhere?

Reply:
yes, usually they are always present under /usr/src/linux/ 
the user than install the own kernel by make && make install
Gentoo is only providing the kernel sources, we don't provide any kernel binary.

Reply:
Once the user has built their kernel, are they free to remove /usr/src/linux (or via the Gentoo package manager?)  If so, providing the emerge line(s) could be helpful.

On a related note, there was a recent enhancement: 1950c193 ("kpatch-build: Add -a, --archversion commandline option") that added the ability to build a kpatch module for a non-running kernel.  It isn't compatible with the `-s|--sourcedir` option, but maybe could be made useful for Gentoo if specific kernel version sources are emerge-able.  This is outside the scope for this PR, but I thought it maybe of interest for the kpatch build-server tool that you are developing.

Reply:
the source code under /usr/src/linux is linked by eselect kernel and is suggested not to remove. But it could have been clean by emerge if the user is moving to a new kernel and after calling the cleaning function. 
elivepatch have a special function for emerge old kernel, because we are keeping only recent kernel in the Gentoo-sources repository tree. (https://github.com/gentoo/gentoo/tree/master/sys-kernel/gentoo-sources)
Maybe I could try adding it to kpatch-build, in a later pull request. That function is using a overlay for getting old kernel. (https://github.com/aliceinwire/gentoo-sources_overlay/tree/master/sys-kernel/gentoo-sources)
something like -g for emerging old Gentoo sources.